src/main/resources/com/google/api/codegen/java/grpc_package.snip

```diff
@@ -0,0 +1,138 @@
+@snippet generate(metadata)
```

Comment:
I did it on purpose. The upload settings of the packman template was inconsistent with other artifacts (e.g. toolkit, gax-java). The packman template uses mavenUsername/mavenPassword but others use ossrhUsername/ossrhPassword. According to the vkit-java playbook, @vam-google encountered this issue as well. I feel this is a good chance to fix this problem.